release/build_release.sh

```diff
@@ -26,6 +26,21 @@ function check_usage {
 		print_help
 	fi
 
+	if [ -z "${LIFERAY_RELEASE_SOFT}" ]
+	then
+		LIFERAY_RELEASE_SOFT="false"
+	fi
+
+	if [ "${LIFERAY_RELEASE_SOFT}" == "true" ] && [ -z "${LIFERAY_RELEASE_TICKETS}" ]
+	then
+		print_help
+	fi
+
+	if [ "${LIFERAY_RELEASE_SOFT}" == "true" ] && [ -z "${LIFERAY_RELEASE_GIT_PREV_REF}" ]
+	then
+		print_help
+	fi
```

Comment:
@kiwm, can we optimize and use a single `if`?

```
if [ "${LIFERAY_RELEASE_SOFT}" == "true" ] && [ -z "${LIFERAY_RELEASE_TICKETS}" ||  -z "${LIFERAY_RELEASE_GIT_PREV_REF}"]
	then
		print_help
	fi
```

Please make sure the syntax around the `||` is correct. :wink: I'm just sharing the idea with you and I'm not sure if the syntax is 100% correct. I mean, I'm not sure this `[ -z "${LIFERAY_RELEASE_TICKETS}" ||  -z "${LIFERAY_RELEASE_GIT_PREV_REF}"]` is correct from a bash syntax perspective.